src/Controller/ContactController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Component\Mime\Email;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use App\Form\ContactType;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Mailer\MailerInterface;
+
+/**
+ * @Route("/contact", name="contact")
+ */
+
+class ContactController extends AbstractController
+{
+
+    public const MAILER = 'swap.wild@gmail.com';
```

Comment:
This should probably be a configuration environnement variable.
See : https://symfony.com/doc/current/configuration.html